internal/pkg/configuration/configuration.go

```diff
@@ -53,7 +53,8 @@ func Generate(ctx context.Context, in *machine.GenerateConfigurationRequest) (re
 						DeviceCIDR:      device.Cidr,
 						DeviceDHCP:      device.Dhcp,
 						DeviceIgnore:    device.Ignore,
-						DeviceRoutes:    routes,
+						// TODO: this looks strange, since its always empty de-facto.
```

Comment:
I have no idea whats happening here

Reply:
@Unix4ever should know! I think it should be a copy, but ?